apps/graphql/src/mocks/itinerary.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+const getCountry = () => ({
+  id: 'aaa',
+  name: 'Norway',
+  code: 'NO',
+  slug: '',
+  flagURL: '',
```

Comment:
pls, add some dummy date also here instead of the empty string if it is not necessary.
It will be easiest to recognize that everything is working properly.
It can be just something like that:
```
slug: 'slug',
flagURL: 'flagUrl',
```

Reply:
Done